src/Encrypt/Openssl.php

```diff
@@ -464,4 +471,15 @@ public function toString()
     {
         return 'Openssl';
     }
+
+    /**
+     * Free key resource if necessary.
+     * PHP 8 automatically frees the key instance and deprecates the function
+     */
+    private function freeKeyResource($keys): void
```

Comment:
The DocBlock for `$keys` is missing.

Reply:
true :+1:

Reply:
Please add proper native typehint for `keys` here.